### PR TITLE
Dev #T207: Make it possible to change the base language of a survey

### PR DESCRIPTION
**Pull request by gabrieljenik**

### CODE REVIEW

File: application/extensions/admin/survey/LanguagesWidget/LanguagesWidget.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<?php
+
+class LanguagesWidget extends WhSelect2
```
Comment (olleharstedt) [inline application/extensions/admin/survey/LanguagesWidget/LanguagesWidget.php:3]: Hm, why is a new widget needed for a simple select box? Or did I miss something?

Comment (olleharstedt) [reply application/extensions/admin/survey/LanguagesWidget/LanguagesWidget.php:3]: And why is that needed?

Comment (olleharstedt) [reply application/extensions/admin/survey/LanguagesWidget/LanguagesWidget.php:3]: Ah ok. That's a good answer. :)

Comment (gabrieljenik) [reply application/extensions/admin/survey/LanguagesWidget/LanguagesWidget.php:3]: > Ah ok. That's a good answer. :)

I have my moments :)

**Pull request merged**